docs/analytics_engine.md

```diff
@@ -0,0 +1,53 @@
+# ANALYTICS ENGINE (RECOMMENDER)
+
+Analytics Engine provides an analysis of telemetry data and resource utlilization data
+
+## Usage
+
+### API
+
+API description provided here : [Web API Readme](https://github.com/mF2C/analytics_engine/blob/master/analytics_engine/heuristics/sinks/mf2c/README.md)
+
+#### Examples
+
+Please see above referred document for examples 
+
+### Troubleshooting
+
+If incorrect telemetry data is returned or errors, please check and make sure the host name is exported as HOSTNAME environment variable.
+
+## CHANGELOG
+
+### 10.05.2019 (date day/month/year)
```

Comment:
you're missing the component version here

Reply:
Where exactly should the version be mentioned?

Reply:
Added version number now